src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,210 @@
+                                />
+                            <label >Shirt</label>
+                        </div>
+                        <div>
+                            <input 
+                                type="checkbox" 
+                                id="pant" 
+                                name="pant" 
+                                value="pant"  
+                                onChange={() => clickHandler("pant")}
+                                checked={byCategoryNames.includes("pant")}
+                                />
+                            <label >Pant</label>
+                        </div>
+                        <div>
+                            <input 
+                                type="checkbox" 
+                                id="shoes" 
+                                name="shoes" 
+                                value="shoes"  
+                                onChange={() => clickHandler("shoes")}
+                                checked={byCategoryNames.includes("shoes")} 
+                            />
+                            <label >Shoes</label>
+                        </div>
+                        <div>
+                            <input 
+                                type="checkbox" 
+                                id="glass" 
+                                name="glass" 
+                                value="glass"   
+                                onChange={() => clickHandler("goggles")}
+                                checked={byCategoryNames.includes("goggles")}
+                                />
+                            <label > Goggles</label>
+                        </div>
+                    </div>
+                    {/* <!-- end of category --> */}
+                    <p className="spacer"></p>
+                    {/* <!-- rating --> */}
+                    <h4>Rating</h4>
+                    <hr />
+                    <div className="filter-rating">
+                        <div id="rating-in-line">
+                            <p>1<span class="fa fa-star checked"></span></p>
+                            <p>3<span class="fa fa-star checked"></span></p>
+                            <p>5<span class="fa fa-star checked"></span></p>
+                        </div>
+                        <input 
+                            className="range-selector" 
+                            type="range" 
+                            min="1" 
+                            max="5"
+                            step="2"
+                           
+                            onChange={(e) => filterDispatch({ type: "FILTER_BY_RATING", payload: Number(e.target.value) })}
+                                // onChange={(e)=>value(e.target.value)}
+                            />
+                    </div>
+
+                    {/* <!-- end of rating --> */}
+                    <p className="spacer"></p>
+
+                    {/* <!-- price --> */}
```

Comment:
Appreciation: good way of using comment line to give message like where what  start is starting  and what is  ending . I will also use in my code . Great

Reply:
Thank you, Shubham
Yes, i use the comment method consistently.